markdown/dev/howtos/git/editing-on-github/en.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Editing freesewing content on Github
+---
+
+## Introduction
+This 'How to' is designed to help get you started editing content using the [Github website](https://github.com/freesewing) where you can edit pattern instructions and website pages.  For info, it is also possible to make changes using the Git desktop application and/or your computer's command line; these advanced topics are beyond the scope of this 'How to'.
+
+Git allows Freesewing supporters to suggest changes and helps the administrators to review the code and implement approved changes. If you are new to Git, you will need to [create an account](https://github.com/signup?user_email=&source=form-home-signup).
+
+All pattern instructions and web pages (including this 'How to') are written using Markdown code. Please see our [Markdown guide](https://freesewing.dev/guides/markdown) for more information on this.
```

Comment:
I prefer `written in markdown` to `written using markdown code` because the latter sounds more scary than it is ;)